### ISSUE TITLE
Dragging bookmark causes it to disappear into another folder.

### ISSUE THREAD

**Issue by joshkerr** [conceptualspace/yet-another-speed-dial#83]
I sometimes accidentally drag a bookmark instead of clicking on it. As a result, the bookmark disappears into another folder. This is annoying.

I suggest either checking to see if the dragged bookmark is dropped on a destination folder, or showing a confirmation before moving the bookmark.

**Comment by pejotigrek** [comment]
same here, I've also noticed that after this disappearing/moving-out-uncontrollably I can't edit this one bookmark [changing the title doesn't do anything], and sometimes it randomly tries to refresh it's thumbnail by opening new tab and then succeeding on that.

I use drag-n-drop to sort my bookmarks by hand so I can have them the way I find more efficient to use certain bookmarks, so this issue drives me crazy when making some changes ;)

**Comment by conceptualspace** [comment]
yes, the bookmark should only move folders if you hover over a folder title, or if you pause while over the folder title, that folder should load so you can select where in the new folder to drop it. i think it know the source of this bug, will patch in next release

**Comment by loopernow** [comment]
This happened to me when I was trying to move a bookmark from the top of the group to the bottom.
